lightningd/plugin_control.c

```diff
@@ -133,9 +144,21 @@ static struct command_result *plugin_start(struct dynamic_plugin *dp)
 	/* Give the plugin 20 seconds to respond to `getmanifest`, so we don't hang
 	 * too long on the RPC caller. */
 	p->timeout_timer = new_reltimer(dp->cmd->ld->timers, dp,
-	                                time_from_sec((10)),
+	                                time_from_sec((20)),
```

Comment:
Wasn't 10 seconds long enough ?

Reply:
The documentation says 20 seconds, and this was not matching, so I adapted the value to the documented one. We can also go the other way around :-)